repositories/src/main/scala/cromwell/pipeline/datastorage/DatastorageModule.scala

```diff
@@ -20,6 +21,7 @@ class DatastorageModule(applicationConfig: ApplicationConfig) {
     new UserRepository(pipelineDatabaseEngine, databaseLayer)
   lazy val projectRepository: ProjectRepository =
     new ProjectRepository(pipelineDatabaseEngine, databaseLayer)
+  lazy val womTool: WomTool = new WomTool
```

Comment:
Are you sure that `WomTool` should be created in `Datastorage` module?

Reply:
I don't know about can we count `WomToo`l as Service, so I decided to put it into `Datastorage`

Reply:
Well, it is definitely has nothing to do with `DatastorageModule` :)
`womtool` is a separate module, so I think you can create `WomtoolModule` class where you will initialize `Womtool` and then pass it to `ServiceModule`

Reply:
Yes, did it, thanks